src/pybind/mgr/dashboard/controllers/multi_cluster.py

```diff
@@ -101,6 +105,26 @@ def auth(self, url: str, cluster_alias: str, username: str,
 
         return False
 
+    def get_cors_endpoints_string(self, hub_url):
+        parsed_url = urlparse(hub_url)
+        hostname = parsed_url.hostname
+        cors_endpoints_set = set()
+        cors_endpoints_set.add(hub_url)
+
+        orch = OrchClient.instance()
+        inventory_hosts = [host.to_json() for host in orch.hosts.list()]
+
+        for host in inventory_hosts:
+            host_addr = host['addr']
+            host_ip_url = hub_url.replace(hostname, host_addr)
+            host_hostname_url = hub_url.replace(hostname, host['hostname'])
```

Comment:
this will break ceph-dev like environments right? where no orchestrator is available?

Reply:
@nizamial09 , works fine with ceph-dev as well